<ui version="4.0" >
 <class>lp_ladspa_slider_float</class>
 <widget class="QWidget" name="lp_ladspa_slider_float" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>75</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="2" column="0" colspan="2" >
    <widget class="QDoubleSpinBox" name="sp_val" />
   </item>
   <item row="1" column="1" >
    <widget class="QwtSlider" name="sl_val" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="scalePosition" >
      <enum>QwtSlider::NoScale</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QPushButton" name="pb_def_val" >
     <property name="maximumSize" >
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
     <property name="text" >
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="txt_name" >
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtSlider</class>
   <extends>QWidget</extends>
   <header>qwt_slider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
